### PR TITLE
[ONNX] export sum, prod, sqrt improve log_softmax.

### PR DESCRIPTION
**Pull request by sighingnow**
More exports:
+ `sum`
+ `prod`
+ `sqrt`

Improve:
+ `log_softmax`

Doc:
+ Fix a URL typo in `docs/source/onnx.rst`

### CODE REVIEW

**Comment by houseroad** [comment]
Could you also add the onnx tests in https://github.com/onnxbot/onnx-fb-universe/blob/master/test/test_operators.py?

**Comment by ezyang** [comment]
@pytorchbot test this please

This is so legit! Thank you!

**Comment by ezyang** [comment]
The test output needs updating. It's as simple as running `python test/test_jit.py --accept`.

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -170,6 +170,10 @@ def neg(g, self):
     return g.op("Neg", self)
 
 
+def sqrt(g, self):
+    return g.op('Sqrt', self)
```
Comment (ngimel) [inline torch/onnx/symbolic.py]: nit: should not it be double quotes, like all other operations?

Comment (sighingnow) [reply torch/onnx/symbolic.py]: Done.

**Comment by sighingnow** [comment]
Update trace log for `log_softmax`.

**Comment by sighingnow** [comment]
Update axis_i to "list of ints", according to the onnx's doc: https://github.com/onnx/onnx/blob/master/docs/Operators.md#attributes-48

**Comment by sighingnow** [comment]
Onnx test done in onnxbot/onnx-fb-universe#247

/cc: @houseroad

**Comment by sighingnow** [comment]
Line ends fixed.

**Comment by soumith** [comment]
thank you @sighingnow !

**Pull request merged**